core-java/src/test/java/org/baeldung/java/diamond/DiamondOperatorTest.java

```diff
@@ -0,0 +1,13 @@
+package org.baeldung.java.diamond;
+
+import static org.junit.Assert.assertNotNull;
+
+import org.junit.Test;
+
+public class DiamondOperatorTest {
+    @Test
+    public void shouldCreateCarUsingDiamondWithoutWarnings() {
```

Comment:
Not a very interesting test...but let's at least follow the naming convention for tests: givenX_whenY_thenZ (the givenX is optional). So you would have something like: whenCreateCarUsingDiamondOperator_thenSuccess

Reply:
Renamed. I know it's not a very interesting test, but taking into account the article...

Reply:
That's fair.